google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStubSettings.java

```diff
@@ -515,6 +571,25 @@ private Builder() {
                               .build())
                       .build());
 
+      long maxElementPerBatch = 100L;
+      bulkReadRowsSettings =
+          BigtableBulkReadRowsCallSettings.newBuilder(new ReadRowsBatchingDescriptor())
+              .setRetryableCodes(readRowsSettings.getRetryableCodes())
+              .setRetrySettings(IDEMPOTENT_RETRY_SETTINGS)
+              .setBatchingSettings(
+                  BatchingSettings.newBuilder()
+                      .setIsEnabled(true)
+                      .setElementCountThreshold(maxElementPerBatch)
+                      .setRequestByteThreshold(400L * 1024L)
```

Comment:
can we make this a variable?

Reply:
We could have a variable here. Although as in this format, it is similar to the existing `bulkMutateRowsSettings` configuration
https://github.com/googleapis/java-bigtable/blob/abbc6c8c0ffbe1dcf88c14b1e70fe3150c61c582/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStubSettings.java#L500-L516

We might need to change that to have common formatting in separate PR